File: app/src/main/java/org/stepic/droid/web/SubmissionRequest.java
package org.stepic.droid.web;

import org.stepic.droid.model.Reply;
import org.stepic.droid.model.Submission;

public class SubmissionRequest {
    Submission submission;

    public SubmissionRequest(Reply reply, long attemptId) {
        submission = new Submission(reply, attemptId);
    }
}

File: app/src/main/java/org/stepic/droid/web/SubmissionRequest.kt
package org.stepic.droid.web

import org.stepik.android.model.Submission
import org.stepik.android.model.Reply

class SubmissionRequest(val submission: Submission) {
    constructor(reply: Reply, attemptId: Long) : this(Submission(reply = reply, attempt = attemptId))
}